src/main/java/model/calculator/Calculator.java

```diff
@@ -0,0 +1,90 @@
+package model.calculator;
+
+import java.util.ArrayList;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import model.winningLottery.Lottery;
+import model.winningLottery.Ranking;
+
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.*;
+import static model.Amount.LOTTO_UNIT_PRICE;
+
+public class Calculator {
+    private Map<Ranking, Integer> rankingCountMap = new EnumMap<>(Ranking.class);
+    private List<LottoResult> lottoResultList = new ArrayList<>();
+
+    public Calculator(List<LottoResult> lottoResultList) {
+        this.lottoResultList = lottoResultList;
+        initMap();
```

Comment:
```suggestion
        this.rankingCountMap = initMap();
```
반환 값을 통해서 생성자에서 초기화를 하면 상태값이 초기화 되는 느낌을 바로 받을 수 있을 것 같아요